internal/backend/deployments/deployments.go

```diff
@@ -53,25 +53,25 @@ func New(l *zap.Logger, cfg *Config, db db.DB, telemetry *telemetry.Telemetry) s
 }
 
 func (d *deployments) Autodrain() {
-	d.l.
-		With(zap.Duration("interval", d.cfg.AutoDrainingDeactivationInterval),
-			zap.Duration("jitter", d.cfg.AutoDrainingDeactivationJitter)).
-		Info("periodically autoinactivating drained deployments")
+	l := d.l.With(
```

Comment:
i don't think this should appear in every log message

Reply:
done